### ISSUE TITLE
pngquant --speed does not do anything

### ISSUE THREAD

**Issue by dbaronov** [1000ch/gulp-image#48]
I have this config:

```
	.pipe(image({
		pngquant: true,
		optipng: false,
		zopflipng: false,
		jpegRecompress: true,
		mozjpeg: false,
		guetzli: false,
		gifsicle: false,
		svgo: true,
		concurrent: 10,
		options: {
			pngquant : ['--speed=10'],
		}
	}))
```

Unless I am missing something, the **pngquant : ['--speed=10'],** does not do anything, as processing speed is the same as without this setting.

Any ideas?

**Comment by 1000ch** [comment]
@dbaronov Try this, and fixed confusing usage (d28f57a).

```javascript
.pipe(image({
  pngquant: ['--speed=10'],
  optipng: false,
  zopflipng: false,
  jpegRecompress: true,
  mozjpeg: false,
  guetzli: false,
  gifsicle: false,
  svgo: true,
  concurrent: 10
}))
```

**Comment by 1000ch** [comment]
@dbaronov 

> still does not affect a speed too much as only 1 - 2 seconds faster.

This is not a problem of this plugin, but pngquant itself 😅
